File: Java/005.java
public class Problem005 {

    // Function to find the greatest common divisor of two numbers
    public static long gcd(long a, long b) {
        while (b > 0) {
            long temp = b;
            b = a % b;
            a = temp;
        }
        return a;
    }

    // Function to find the least common multiple of two numbers
    public static long lcm(long a, long b) {
        return (a * b) / gcd(a, b);
    }

    public static void main(String[] args) {
        long result = 1;
        for (long i = 1; i <= 20; i++) {
            result = lcm(result, i);
        }
        System.out.println(result);
    }
}

File: Rust/005.rs
fn gcd(a: u64, b: u64) -> u64 {
    if b == 0 {
        a
    } else {
        gcd(b, a % b)
    }
}

fn lcm(a: u64, b: u64) -> u64 {
    (a * b) / gcd(a, b)
}

fn main() {
    let mut result: u64 = 1;
    for i in 1..=20 {
        result = lcm(result, i);
    }
    println!("{}", result);
}
